sebs/local/config.py

```diff
@@ -24,14 +26,27 @@ def deserialize(config: dict, cache: Cache, handlers: LoggingHandlers) -> Creden
 class LocalResources(Resources):
     def __init__(self, storage_cfg: Optional[MinioConfig] = None):
         super().__init__(name="local")
+        self._path: str = ""
         self._storage = storage_cfg
+        self._allocated_ports = set()
 
     @property
     def storage_config(self) -> Optional[MinioConfig]:
         return self._storage
 
+    @property
+    def path(self) -> str:
```

Comment:
What is the point of path? I don't see it being used anywhere? I also don't see it being assigned any non-null value?

Reply:
we're using it in `LocalConfig`'s (class in `sebs/local/config.py`) serialize method to store allocated ports.